tests/components/MCQTagOptions.test.ts

```diff
@@ -28,12 +28,7 @@ describe("MCQTagOptions.vue", () => {
     const tags = [];
 
     const uniqueCourses = getUniquePropertyValues(tags);
-    expect(uniqueCourses).toEqual({
-      course: [],
-      subject: [],
-      system: [],
-      animal: [],
-    });
+    expect(uniqueCourses).toEqual({});
```

Comment:
Because the `uniqueCourses` are populated based on how many taxonomies there are instead of having a static and definite amount. So it will initially be empty then populated later.